docs/reading_events.md

```diff
@@ -0,0 +1,151 @@
+# Reading Events
+
+## Reading from a specific stream
+
+The simplest way to read a stream forwards is to supply a `PgEventstore::Stream` object.
+
+```ruby
+stream = PgEventstore::Stream.new(context: 'MYAwesomeContext', stream_name: 'User', stream_id: '1')
+PgEventstore.client.read(stream)
+# => [#<PgEventstore::Event 0x1>, #<PgEventstore::Event 0x1>, ...]
+```
+
+### max_count
+
+You can provide the `:max_count` option. This option determines how much records to return in a response. Default is `1000` and it can be changed via `:max_count` configuration setting(see [**"Configuration"**](configuration.md) chapter):
+
+```ruby
+stream = PgEventstore::Stream.new(context: 'MYAwesomeContext', stream_name: 'User', stream_id: '1')
+PgEventstore.client.read(stream, options: { max_count: 100 })
+```
+
+### resolve_link_tos
+
+When using projections(Event Sourcing Projections are meant here) to create new events you can set whether the generated events are pointers to existing events. Setting this value to `true` tells `pg_eventstore` to return the original event instead "link" event.
```

Comment:
```suggestion
When reading streams with projected events (links to other events) you can chose to resolve those links by setting `resolve_link_tos`to `true`, returning the original event instead of the "link" event.
```